apps/aering/src/aer_compiler.erl

```diff
@@ -5,7 +5,8 @@
 %%%     Compiler from Aeterinty Ring language to the Aeternity VM, aevm.
 %%% @end
 %%% Created : 12 Dec 2017
-%%% aec_conductor:stop_mining(),aer_compiler:file(a,c).
+%%% aec_conductor:stop_mining().
+%%% aer_compiler:file( identity, [pp_ast,pp_icode,pp_assembler,pp_bytecode, pp_ring_code]).
```

Comment:
Should this be here? If so maybe add a short explanation.

Reply:
Temporary stuff to run from the command shell.. I'll leave it in for now.